app/scripts/controllers/network/network-controller.test.ts

```diff
@@ -4137,16 +4127,43 @@ describe('NetworkController', () => {
           },
         },
         async ({ controller }) => {
-          const network = new CustomNetworkCommunications({
```

Comment:
The new version of this test now seems very similar to existing tests for `rollbackToPreviousProvider` (e.g. line 5865). Perhaps we don't need this test anymore?

Reply:
Good point. Yeah that would make sense to me, I'll remove both of these that you've commented on.

Reply:
Removed in 2ea6a0476a7634ce97d4bef938ed7c1fc5198a67